src/client/search/SearchActions.js

```diff
@@ -22,3 +25,24 @@ const addQueryParamTypes = (searchQuery, query, paramTypes) => {
     paramTypes,
   };
 };
+
+export const INVALIDATE_FEED_RESULTS = 'INVALIDATE_FEED_RESULTS';
```

Comment:
Why is this called INVALIDATE_FEED_RESULTS, I don't understand what you mean by it. 

How about something like FETCH_SEARCH_RESULTS? or any other suggestions you got ?